core/src/block_sync.rs

```diff
@@ -366,16 +367,14 @@ pub mod message {
         /// Send this message over the network to the specified `peer`.
         #[iroha_futures::telemetry_future]
         #[log("TRACE")]
-        pub async fn send_to_peers(self, broker: Broker, peers: &[PeerId]) {
-            let mut futures = Vec::new();
-            for peer in peers {
-                let message = self.clone();
-                let peer = peer.clone();
-                let broker = broker.clone();
-                futures.push(message.send_to(broker, peer));
-            }
-
-            tokio::task::spawn(futures::future::join_all(futures));
+        pub async fn send_to_multiple(self, broker: Broker, peers: &[PeerId]) {
```

Comment:
```suggestion
        pub async fn send(self, broker: Broker, peers: &[PeerId]) {
```
I understand why you removed `peers` (the type of the argument). Since you're sending it to an array of peers, the `multiple` is implied too.

Reply:
I guess one intent here is not the elimination of verbosity but the contrast with `send_to` method